fastlane/actions/rocket.rb

```diff
@@ -0,0 +1,54 @@
+module Fastlane
+  module Actions
+    class RocketAction < Action
+      def self.run(params)
+        puts "
+            /\\
+           |==|
+           |  |
+           |  |
+           |  |
+          /____\\
+          |    |
+          |    |
+          | FL |
+          |    |
+         /| |  |\\
+        / | |  | \\
+       /__|_|__|__\\
```

Comment:
@JaviSoto absolutely, here:

~~~
           ____
          /    \
         |      |
         |      |
         |      |
          \____/
          |    |
          |    |
          |    |
          |____|
         {|    |}
          |    |
          |    |
          | F  |
          | A  |
          | S  |
          | T  |
          | L  |
          | A  |
         /| N  |\
         || E  ||
         ||    ||
         \|____|/
          /_\/_\
          ######
         ########
          ######
           ####
           ####
            ##
            ##
            ##
            ##
~~~

As you can see:

- I added a nose cone to protect your app during flight to orbit
- Also added a beefier first and second stage. Fastlane is serious business, not just for light toy payloads.
- There are now landing legs and grid fins (currently stowed). Imagine how expensive Fastlane would be if you could only use it once.